modules/php/States/PlayCardTrait.php

```diff
@@ -66,6 +87,14 @@ public function arg_playCard()
     if ($playRule == -1) {
       return ["count" => "All but 1"];
     }
+
+    $addInflation = Utils::getActiveInflation() ? 1 : 0;
+    $partyBonus =
+      Utils::getActivePartyBonus() && Utils::isPartyInPlay()
+        ? 1 + $addInflation
+        : 0;
+    $playRule += $addInflation + $partyBonus;
+
     return ["count" => $playRule - $played];
```

Comment:
Need to add the rich bonus here. Will do.

Reply:
It is actually even more complex no that I realize:
every time any keeper card is played or moved around or discarded, we need to check again if the Rich or Poor bonus apply to the current player with the new situation of keepers.
Unless they already received it during their turn, because they can only get it once - so we probably need game states for each, that are reset on the next player turn.

Maybe I would suggest to merge & include this branch as it is, to have as much as possible in the alfa release.
And consider this as additional changes in a separate branch for Rich & Poor.

Reply:
OK, I'll add a comment in the code so we keep in mind that this needs to be fixed.